micro-ui/web/packages/modules/workbench/src/components/FinalPopup.js

```
@@ -0,0 +1,55 @@
+import { useTranslation } from "react-i18next";
+
+const FinalPopup = ({ showFinalPopUp, setShowFinalPopup, onConfirmClick})=> {
+    const { t } = useTranslation();
+    return (
+        showFinalPopUp &&  (
+            <PopUp 
+                className={"custom-popup"}
+                type={"alert"}
+                alertMessage={t("YOU_WON'T_BE_ABLE_TO_UNDO_THIS_STEP_OF_CREATING_HIERARCHY")}
+                alertHeading={t("CREATE_BOUNDARY_HIERARCHY")}
+                children={[
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)